api/nodes/serializers.py

```diff
@@ -15,15 +15,18 @@ class NodeSerializer(JSONAPISerializer):
     date_modified = ser.DateTimeField(read_only=True)
 
     links = LinksField({
```

Comment:
@samanehsan @brianjgeiger What are your thoughts on this API (`LinksField` + `Link` + method names)? This was a proof-of-concept proposed as an alternative to the `LinksSerializer` + `get_links` API implemented in https://github.com/CenterForOpenScience/osf.io/commit/dceda7301884b6f7d2ba1c1096f17d612366acaa .

Not sure which is better, so I welcome your feedback.